@navikt/core/react/src/modal/dialog-polyfill.ts

```diff
@@ -632,6 +632,7 @@ dialogPolyfill.DialogManager = function () {
       var removed = [];
       records.forEach(function (rec) {
         // biome-ignore lint/suspicious/noImplicitAnyLet: Reduntant to type c in this scenario
+        // biome-ignore lint/suspicious/noAssignInExpressions: <explanation>
```

Comment:
Should we just add this file to the ignore list instead?

Reply:
Sounds like a good idea 👍

Reply:
Done 🚀